src/luxor_overrides.jl

```diff
@@ -148,3 +148,89 @@ function scaleto(x, y)
     Luxor.scale(scaling...)
     cs.current_scale = (x, y)
 end
+
+function animate_text(
+    str,
+    pos::Point,
+    valign::Symbol,
+    halign::Symbol,
+    angle::Float64,
+    t::Float64,
+)
```

Comment:
Missing a docstring here -- I see a lot of the code came directly from Luxor. It seems like there is enough of a change to merit a docstring for the user.

Reply:
This is an internal function though. I'll add a short developer docstring.